plugins/lookup/bitwarden.py

```diff
@@ -228,7 +272,24 @@ def run(self, terms=None, variables=None, **kwargs):
         if not terms:
             terms = [None]
 
-        return [_bitwarden.get_field(field, term, search_field, collection_id, organization_id) for term in terms]
+        if collection_name and collection_id:
+            raise AnsibleOptionsError("'collection_name' and 'collection_id' are mutually exclusive!")
+        elif collection_name:
+            collection_ids = _bitwarden.get_collection_ids(collection_name, organization_id)
+            if not collection_ids:
+                raise BitwardenException("No matching collections found!")
+        else:
+            collection_ids = [collection_id]
+
+        results = [_bitwarden.get_field(field, term, search_field, collection_id, organization_id) for collection_id in
+                   collection_ids for term in terms]
```

Comment:
As a matter of improving the readability of this:
```suggestion
        results = [
            _bitwarden.get_field(field, term, search_field, collection_id, organization_id) 
            for collection_id in collection_ids 
            for term in terms
        ]
```